resources/clj-kondo.exports/tensegritics/clojuredart/hooks/flutter.clj

```diff
@@ -3,6 +3,17 @@
             [clojure.set :as set]
             [clojure.string :as str]))
 
+(defn- with->bindings
+  "Get :with form, returns mapcat-ed bindigs"
```

Comment:
I guess I will fix this doc to be like: "returns :with bindings vector, :let included'